collect_app/src/androidTest/java/org/odk/collect/android/feature/instancemanagement/SendFinalizedFormTest.kt

```diff
@@ -121,31 +118,4 @@ class SendFinalizedFormTest {
             .clickOnText("One Question")
             .assertOnPage()
     }
-
-    @Test
-    fun whenGoogleUsedAsServer_sendsSubmissionToSheet() {
```

Comment:
I think we should keep this test and use `copyInstance` (like in `InstancesAdbTest`) to set up the submission instead of deleting it.